components/exercises/chapter-4-oscar/Utils.test.tsx

```diff
@@ -30,3 +35,40 @@ describe('arraySum', () => {
     expect(arraySum([-1, -2, -3, -4, -5])).toEqual(-15);
   });
 });
+
+describe('getReverseArray', () => {
```

Comment:
Can we add a test to make sure the array returned is not the same array as the one passed? (so it is a copy)

Reply:
test add and passed